Getting error "Cannot find module"

Trying to use the transformer in my project and constantly getting this error:

```
Error: Cannot find module '.../projects/fabric/src/fab_types'
Require stack:
- .../projects/fabric/node_modules/io-ts-transformer/transformer.js
- .../projects/fabric/node_modules/ttypescript/lib/PluginCreator.js
- .../projects/fabric/node_modules/ttypescript/lib/patchCreateProgram.js
- .../projects/fabric/node_modules/ttypescript/lib/loadTypescript.js
- .../projects/fabric/node_modules/ttypescript/lib/tsc.js
- .../projects/fabric/node_modules/ttypescript/bin/tsc
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:1029:15)
    at Function.resolve (internal/modules/cjs/helpers.js:79:19)
    at getImportNodeRealPath (.../projects/fabric/node_modules/io-ts-transformer/transformer.js:189:20)
    at .../projects/fabric/node_modules/io-ts-transformer/transformer.js:193:63
    at Array.some (<anonymous>)
    at .../projects/fabric/node_modules/io-ts-transformer/transformer.js:58:38
    at transformSourceFileOrBundle (.../projects/fabric/node_modules/typescript/lib/typescript.js:76501:57)
    at transformation (.../projects/fabric/node_modules/typescript/lib/typescript.js:94068:24)
    at transformRoot (.../projects/fabric/node_modules/typescript/lib/typescript.js:94089:82)
    at Object.map (.../projects/fabric/node_modules/typescript/lib/typescript.js:551:29) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '.../projects/fabric/node_modules/io-ts-transformer/transformer.js',
    '.../projects/fabric/node_modules/ttypescript/lib/PluginCreator.js',
    '.../projects/fabric/node_modules/ttypescript/lib/patchCreateProgram.js',
    '.../projects/fabric/node_modules/ttypescript/lib/loadTypescript.js',
    '.../projects/fabric/node_modules/ttypescript/lib/tsc.js',
    '.../projects/fabric/node_modules/ttypescript/bin/tsc'
  ]
}
```

--- a/transformer.ts
+++ b/transformer.ts
@@ -349,7 +349,18 @@ function getImportNodeRealPath(node: ts.ImportDeclaration): string {
     ? path.resolve(path.dirname(node.getSourceFile().fileName), module)
     : module
 
-  return require.resolve(nodePath)
+  try {
+    return require.resolve(nodePath)
+  } catch(e) {
+    // in some cases it fails to resolve modules unless their file extension is included.
+    for (const extension of ['ts', 'tsx']) {
+        try {
+            return require.resolve(`${nodePath}.${extension}`)
+        } catch(e) { }
+    }
+    //if none of the extensions worked
+    throw e
+  }
 }
 
 

